src/inputs/ChipSelectField.tsx

```diff
@@ -102,43 +104,49 @@ export function ChipSelectField<O, V extends Value>(
     getKey: (item) => (isListBoxSection(item) ? item.title : getOptionValue(item)),
   });
 
```

Comment:
question: Not sure I understand the difference/benefits of doing this approach with `!firstRender` vs:
```
const memoOptions = useMemo(() => { title: "Options", options }), [options]);
useEffect(() => listData.update("Options", memoOptions), [memoOptions]);
```

Reply:
thanks for clarifying 👍